src/Settings/appsettings.json

```diff
@@ -7,6 +9,8 @@
   },
 
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=Settings;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "SqlServer": "Server=(localdb)\\mssqllocaldb;Database=Settings;Trusted_Connection=True;MultipleActiveResultSets=true",
+    "Postgres": "User ID=postgres;Password=postgres;Host=localhost;Port=5432;Database=Settings;",
```

Comment:
I know this is for development purposes, but could be a good opportunity to start scoping out user-secrets/app-secrets and storing such credentials in safe ways. 

https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?tabs=visual-studio-code

Reply:
Added #25